modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -227,6 +228,19 @@ Simultaneous multithreading (SMT) is enabled by default. If SMT is not enabled i
 ====
 If you disable `hyperthreading`, whether in the BIOS or in the `install-config.yaml`, ensure that your capacity planning accounts for the dramatically decreased machine performance.
 ====
+endif::ibm-z,ibm-z-kvm[]
+ifdef::ibm-z,ibm-z-kvm[]
++
+[NOTE]
+====
+Simultaneous multithreading (SMT) is enabled by default. If SMT is not available on your {product-title} nodes, the `hyperthreading` parameter has no effect.
+====
++
+[IMPORTANT]
+====
+If you disable `hyperthreading`, whether on your {product-title} nodes or in the `install-config.yaml`, ensure that your capacity planning accounts for the dramatically decreased machine performance.
```

Comment:
```suggestion
If you disable `hyperthreading`, whether on your {product-title} nodes or in the `install-config.yaml` file, ensure that your capacity planning accounts for the dramatically decreased machine performance.
```

> Follow the file name, file name extension, or file type with the word file,

Reply:
I'm also adding "file" in line 229 to be consistent